x/adminmodule/client/cli/tx_submit_proposal.go

```diff
@@ -79,7 +79,7 @@ $ %s tx adminmodule submit-proposal --title="Test Proposal" --description="My aw
 
 			content, _ := govv1types.ContentFromProposalType(proposal.Title, proposal.Description, proposal.Type)
 
-			msg, err := types.NewMsgSubmitProposal(content, clientCtx.GetFromAddress())
+			msg, err := types.NewMsgSubmitProposalLegacy(content, clientCtx.GetFromAddress())
```

Comment:
not legacy proposal submission won't be available via cli anymore. why?

Reply:
because it is basically useless: the only way neutron accepts proposal submission (both legacy & non legacy) is via DAO

Reply:
do we need the submit legacy proposal at all? aren't we safe to remove it?

Reply:
we are not safe to remove it due some modules hasn't updated for sdk-47